src/renderer/components/+workloads-deployments/deployment-scale-dialog.scss

```diff
@@ -20,11 +20,17 @@
     }
 
     .desired-scale {
-      flex: 1 0;
+      flex: 1.1 0;
     }
 
     .slider-container {
-      flex: 1.3 0;
+      flex: 1 0;
+    }
+
+    .plus-minus-container {
+      margin-left: 15px;
+      // for vertical align with .slider-container
+      margin-top: -4px;
```

Comment:
This can be removed if added following styles to `slider-container` div - `flex align-center`.

We're heavily using `flex.box` (https://www.npmjs.com/package/flex.box) library class names for layout stuff.

Reply:
removed in https://github.com/lensapp/lens/pull/1191/commits/f5e5baf5f2d6d069e27a7b9ab9483cec8d43f2f6